paf-mvp-core-js/src/express/utils.ts

```diff
@@ -74,3 +74,17 @@ export const getPayload = <T>(req: Request): T => {
   // See https://stackoverflow.com/questions/37668282/unable-to-fetch-post-without-no-cors-in-header
   return JSON.parse(req.body as string) as T;
 };
+
+/**
+ * Escape a string to be used in a regular expression.
+ * Stolen from https://stackoverflow.com/questions/3446170/escape-string-for-use-in-javascript-regex#answer-6969486
+ * @param stringForRegex
```

Comment:
OPT: Providing an example in the docs would help to understand without checking the stackoverflow.

Reply:
If that's ok with you, I'll handle it in a dedicated PR